zha/application/platforms/sensor/__init__.py

```diff
@@ -198,6 +202,12 @@ def on_add(self) -> None:
                 self.handle_cluster_handler_attribute_updated,
             )
         )
+        if (
+            hasattr(self._cluster_handler, "description")
+            and self._cluster_handler.description is not None
+        ):
+            self._attr_translation_key = None
+            self._attr_fallback_name: str = self._cluster_handler.description
```

Comment:
Having this in the base `Sensor` class feels a bit weird. The `description` attribute will only be available on the `AnalogInputClusterHandler` used for `AnalogInputSensor`.
So, shouldn't we just move this part there (and remove the `hasattr` check as well)?